bundle/manifests/ptp.openshift.io_ptpoperatorconfigs.yaml

```diff
@@ -44,9 +44,9 @@ spec:
                   type: string
                 type: object
               plugins:
-                items:
-                  type: string
-                type: array
+                additionalProperties:
+                  x-kubernetes-preserve-unknown-fields: true
```

Comment:
I am not sure what implication will have in future upgrades or using conversion webhook.
Is this safe to use ? any implications ?